composer.json

```diff
@@ -11,6 +11,15 @@
     "require": {
         "marco-c/wp_serve_file": "dev-master"
     },
+    "require-dev": {
+        "stevegrunwell/wp-enforcer": "^0.4.1"
+    },
+    "post-install-cmd": [
+    	"wp-enforcer"
+	],
+	"post-update-cmd": [
```

Comment:
Please, indent properly here and, again, this should be relative paths `./vendor/bin/wp-enforcer`.